gcs/src/unit_tests/gcs_proto_test.cpp

```diff
@@ -52,7 +52,7 @@ START_TEST (gcs_proto_test)
 
     // write action to the buffer, it should not fit
     char* frg = static_cast<char*>(const_cast<void*>(frg_send.frag));
-    int cnt = std::min(strlen(frg)+1, frg_send.frag_len);
```

Comment:
frg points to garbage. We want to send the string pointed by act_send_ptr.
This was resulting it this test to fail, because strlen(frg) in my case was 4 instead of 20 or so (the proper strlen of the message to send)